### PR TITLE
Add hacker handles

### PR DESCRIPTION
**Pull request by bellesea**

### CODE REVIEW

File: pages/team.js [hunk 1]
```diff
@@ -46,20 +46,26 @@ export default function Team() {
...
+                link="https://github.com/zachlatta"
                 text="Zach dropped out of high school after his freshman year to work in the technology industry and had over 5 million people using his software by the time he turned 17. He founded Hack Club to build the program he wish he had in high school and has been awarded the Thiel Fellowship and Forbes 30 Under 30 for his work."
                 pronouns="he/him"
               />
               <Bio
                 img="/team/christina.jpg"
                 name="Christina Asquith"
                 teamRole="COO"
+                handle="christina"
+                link="https://christina.cool/"
```
Comment (exu3) [inline pages/team.js:59]: Does this link work? It doesn't seem to work for me

File: pages/team.js [hunk 2]
```diff
@@ -267,7 +306,39 @@ and participating in academic competitions with his friends.`}
...
+              variant="title"
+              color="orange"
+              sx={{ lineHeight: '1.75em', fontSize: ['42px', 6] }}
+            >
+              Bots
+            </Text>
+            <Grid columns={[1, null, 2]} gap={4}>
+              <Bio
+                name="Scrappy"
+                teamRole="Book Keeper"
```
Comment (exu3) [inline pages/team.js]:
```suggestion
                teamRole="Bookkeeper"
```

File: pages/team.js [hunk 3]
```diff
@@ -267,7 +306,39 @@ and participating in academic competitions with his friends.`}
...
+              />
+              <Bio
+                name="Orpheus"
+                teamRole="Assistant"
+                text="The bot that helps the team manage everything."
+                img="/team/orpheus.png"
+              />
+              <Bio
+                name="Toriel"
+                teamRole="Bouncer"
```
Comment (exu3) [inline pages/team.js:331]: I'm not sure that Bouncer is appropriate here? It sort of has a strange connotation. Thoughts on changing it to "Greeter"?

File: pages/team.js [hunk 4]
```diff
@@ -362,6 +433,12 @@ and participating in academic competitions with his friends.`}
                 text="Amogh is all about having fun. Whether it’s running an art showcase on the Slack or massive Kahoots at hackathons, he loves to run awesome events. Amogh joined HQ to run spectacular community events, to make Hack Club the best place to be a teenager on the internet, and as Hack Club’s second best rapper."
                 pronouns="he/him"
               />
+              <Bio
+                name="Clippy"
+                teamRole="Community"
+                text="A friendly bot that assists new members with joining Hack Club."
+                img="/team/clippy.jpg"
+              />
```
Comment (exu3) [inline pages/team.js:441]: Could we put Clippy in the Bots section but mention that they are retired? It feels weird to have it in the same section as human staff.

File: pages/team.js [hunk 5]
```diff
@@ -267,7 +306,39 @@ and participating in academic competitions with his friends.`}
...
+              <Bio
+                name="Orpheus"
+                teamRole="Assistant"
+                text="The bot that helps the team manage everything."
+                img="/team/orpheus.png"
+              />
+              <Bio
+                name="Toriel"
+                teamRole="Bouncer"
+                text="Just a lil' greeter bot to direct new members in the Hack Club slack channel and collect some analytics data for that process (and Clippy's killer)."
```
Comment (exu3) [inline pages/team.js]:
```suggestion
                text="What a nice lady. She seems kinda lonely. New Hack Club Slack members are onboarded and welcomed through a series of conversations with Toriel."
```
Suggestion: Make Toriel's bio more like her personality.

Screen Shot 2022-05-29 at 00 17 55

**Review by exu3 — APPROVED** [review]
Thanks for adding this @bellesea! I added a few comments.

I'm not sure about the placement of the handle in between the role and pronouns; it feels a bit out of place there. Thoughts on having it in a muted colour & small font size next to the name instead? Or perhaps just the `mention` icon?

Also, could you run `yarn lint` sometime before this gets merged? Thanks!

###### edit: oops I didn't mean to click the "Approve" button; I think it's looking good so far though!

File: pages/team.js (hunk 1 above)
Comment (bellesea) [reply pages/team.js:59]: > Thanks for adding this @bellesea! I added a few comments.
> 
> I'm not sure about the placement of the handle in between the role and pronouns; it feels a bit out of place there. Thoughts on having it in a muted colour & small font size next to the name instead? Or perhaps just the `mention` icon?
> 
> Also, could you run `yarn lint` sometime before this gets merged? Thanks!
> 
> ###### edit: oops I didn't mean to click the "Approve" button; I think it's looking good so far though!

@exu3 i agree about that the placement could be better... a mock up of next to name or just @ icon. i think the one next to the name would work if all handles were short but some are kinda long and i think it makes that area look even more crowded...maybe we could directly link it to the name?

Screenshot 2022-05-29 at 4 20 16 PM

Screenshot 2022-05-29 at 4 21 27 PM

what are your thoughts on these? (also, just ran yarn lint, thanks!)

File: pages/team.js (hunk 3 above)
Comment (bellesea) [reply pages/team.js:331]: Yes i agree, greeter sounds good! Thanks :))

File: pages/team.js (hunk 5 above)
Comment (bellesea) [reply pages/team.js]: I love this suggestion, thanks!

File: pages/team.js (hunk 4 above)
Comment (bellesea) [reply pages/team.js:441]: yes that makes sense, do you have any ideas for the role? maybe Greeter (retired)?

File: pages/team.js (hunk 1 above)
Comment (exu3) [reply pages/team.js:59]: Ooh linking directly to the name might be a good idea

Thoughts on the layout below, using the `mention` icon (from https://icons.hackclub.com)?
Screen Shot 2022-05-29 at 01 30 14

Comment (bellesea) [reply pages/team.js:59]: ooo i love that idea!
Screenshot 2022-05-29 at 5 02 27 PM

Comment (bellesea) [reply pages/team.js:59]: @sampoder what do you think about this?

File: pages/team.js (hunk 4 above)
Comment (sampoder) [reply pages/team.js:441]: sounds good

File: pages/team.js (hunk 1 above)
Comment (sampoder) [reply pages/team.js:59]: I honestly can't see it, but that might just be my eyes. I think it looks a bit off because the @ normally comes at the start

Comment (sampoder) [reply pages/team.js:59]: But I do prefer the icon over the char

Comment (sampoder) [reply pages/team.js:59]: This design is hard... it feels like things are starting to get cluttered. My fave I've seen is this one:

Comment (bellesea) [reply pages/team.js:59]: yeah i agree the contrast isn't good! what about this? or maybe we can just link the names so we don't need another element...

Screenshot 2022-05-29 at 8 08 48 PM

File: pages/team.js (hunk 4 above)
Comment (bellesea) [reply pages/team.js:441]: done, thank you!

File: pages/team.js (hunk 1 above)
Comment (bellesea) [reply pages/team.js:59]: @sampoder what are your thoughts on the design above^?

**Comment by sampoder** [comment]
Closing due to merge conflicts, feel free to reopen if we still want to pursue this!

**Pull request closed without merging**